exporter/elasticsearchexporter/config.go

```diff
@@ -280,6 +280,9 @@ func (cfg *Config) Validate() error {
 	if cfg.Retry.MaxRetries < 0 {
 		return errors.New("retry::max_retries should be non-negative")
 	}
+	if cfg.Retry.MaxRequests > 0 && cfg.Retry.MaxRetries > 0 {
```

Comment:
```suggestion
	if cfg.Retry.MaxRequests != 0 && cfg.Retry.MaxRetries != 0 {
```

Reply:
:facepalm: I did exactly that but forgot to push